source/includes/steps-fle-convert-to-a-remote-master-key-kmip.yaml

```diff
@@ -0,0 +1,132 @@
+title: Configure Your KMIP KMS
+ref: configure-kmip-kms
+content: |
+
+  To connect a MongoDB driver client to your KMIP KMS, you must configure your KMS
+  such that it accepts your client's TLS certificate.
+
+  Consult the documentation for your KMIP KMS for information on how
+  to accept your client certificate with your specific KMS.
+
+---
+title: Specify your Certificates
+ref: specify-certificates
+content: |
+
+  Your client must connect to your KMIP KMS through TLS and present
+  a client certificate accepted by your KMS server.
+
+  .. tabs-drivers::
+
+     .. tab::
+        :tabid: java-sync
+
+        Specify the following Java system properties to configure your client's
+        TLS connection: 
+
+        .. code-block:: shell
+        
+           -Djavax.net.ssl.enabled=true
+           -Djavax.net.ssl.keyStoreType=pkcs12
+           -Djavax.net.ssl.keyStore=<path to pkcs12 KeyStore containing your client certificate>
+           -Djavax.net.ssl.keyStorePassword=<KeyStore password>
+           -Djavax.net.ssl.trustStoreType=jks
```

Comment:
These three are typically not required so long as the KMS server is using one of the many standard CAs.  For instance, this is not needed to connect with Atlas servers.

Reply:
Removing the 3 trust store related system properties.